### ISSUE TITLE
[  FAILED  ] builtin.log size too small

### ISSUE THREAD

**Issue by vt-alt** [iovisor/bpftrace#2733]
When we run bpftrace' tests we get error:
```
[ RUN      ] builtin.log size too small
[  FAILED  ] builtin.log size too small
        Command: /usr/src/RPM/BUILD/bpftrace-0.16.0/x86_64-alt-linux/src//bpftrace -ve 'BEGIN { if (str($1) == str($2)) { printf("%s\n", str($1)); exit() } }' "hello" "hello"
        Expected: Error loading program: BEGIN
        Found: INFO: node count: 16\nAttaching 1 probe...\n\nError log: \n\nERROR: Error: Failed to load program, verification log buffer not big enough, try increasing the BPFTRACE_LOG_SIZE environment variable beyond the current value of 1 bytes\n
```
This happens on bpftrace 0.15.0 and 0.18.0.

**Comment by viktormalik** [comment]
This is already being addressed by #2731.

**Comment by vt-alt** [comment]
Thanks! I will follow it then.
